modules/ROOT/pages/enabling-api-catalog.adoc

```diff
@@ -0,0 +1,33 @@
+= Enabling API Catalog
+
+Enable a connected Salesforce organization to access and sync Anypoint Platform APIs into the Salesforce API Catalog and expose them as invocable actions. This capability is enabled in Anypoint Platform Access Management by an Anypoint Platform organization administrator.  
+
+[[enable-disable-api-catalog]]
+== Enable or Disable API Catalog in Anypoint Platform
+
+include::partial$include-nav-steps-org-admin.adoc[]
+
+. Click *Salesforce*.
+. In *Capabilities Managed in Anypoint Platform*, click *Settings* for *Make MuleSoft Anypoint Platform APIs Available in Salesforce*.
```

Comment:
Right now these options don't seem to match what's in QAX. Is there a new design for this I should be using for the review?